### PR TITLE
Add Intellisense in Repl, with TAB, ESC, UP, DOWN keys management

### PR DESCRIPTION
**Pull request by LucGenetier**

### CODE REVIEW

File: src/libraries/Microsoft.PowerFx.Repl/Repl.cs [hunk 1]
```diff
@@ -55,7 +56,7 @@ public class PowerFxREPL
         public ParserOptions ParserOptions { get; set; } = new ParserOptions() { AllowsSideEffects = true };
 
         // example override, switching to [1], [2] etc.
-        public virtual string Prompt => "\n>> ";
```
Comment (jas-valgotar) [inline src/libraries/Microsoft.PowerFx.Repl/Repl.cs:58]: >"\n>> [](http://example.com/codeflow?start=40&length=5)

Why this change? does this mean new prompt will now not start on a new line?

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/libraries/Microsoft.PowerFx.Repl/Repl.cs (hunk 1 above)
Comment (LucGenetier) [reply src/libraries/Microsoft.PowerFx.Repl/Repl.cs:58]: Was creating a mess when rewriting the current line :)

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by jas-valgotar — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.
